the-blue-alliance-ios/Frameworks/MyTBAKit/Models/MyTBARegister.swift

```diff
@@ -12,18 +12,20 @@ struct MyTBARegisterRequest: Codable {
 
 struct MyTBARegisterResponse: MyTBAResponse, Codable {
 
-    var code: Int
+    var code: String
     var message: String
 
 }
 
 extension MyTBA {
 
+    @discardableResult
     func register(_ token: String, completion: @escaping (_ error: Error?) -> Void) -> URLSessionDataTask? {
         return registerUnregister("register", token: token, completion: completion)
     }
 
-    func unregister(_ token: String, completion: @escaping (_ error: Error?) -> Void) -> URLSessionDataTask? {
+    @discardableResult
+    @objc func unregister(_ token: String, completion: @escaping (_ error: Error?) -> Void) -> URLSessionDataTask? {
```

Comment:
the @objc is so that we can override this method in our mock, since you can't override a method in an extension in Swift. The real answer here is probably to not mock the myTBA stuff